### PR TITLE
Feature/lu 2632/recombine validation outputs

### PR DESCRIPTION
**Pull request by RyanKilbane**

### CODE REVIEW

File: query_sqs.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class QuerySqs:
+    def __init__(self, name):
+        '''
+        Create AWS resources
+        '''
+        sqs = boto3.resource('sqs')
+        self.queue = sqs.get_queue_by_name(QueueName=name)
+    def get_message(self) -> dict:
+        '''
+        Get a single message off the queuse
```
Comment (mattprice-28) [inline query_sqs.py]: typo in queue

File: test/format_test.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    assert recombined.format_recombinant() == expected_outputs
+
+@mock_s3
+@mock_sqs
+def test_formatting_on_input():
+    connect = boto3.resource('s3', region_name='eu-west-2')
+    connect.create_bucket(Bucket='ValidationOutputs')
+    boto3.client('s3', region_name='eu-west-2').put_object(
+                                                Bucket='ValidationOutputs', 
+                                                Key='12345678900202012066B123-P456-M789QVDQ',
```
Comment (mattprice-28) [inline test/format_test.py]: Is it ok for these keys to be visible?

Comment (RyanKilbane) [reply test/format_test.py]: That's just a test file, the bucket needs to be hard coded since Moto needs to create it.

File: query_sqs.py (hunk 1 above)
Comment (RyanKilbane) [reply query_sqs.py]: It's only a comment, but fixed anyway.

File: recombinator.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    def filtration(self):
+        filtered = Filtration.filtration(self.completed_valids.current_validations)
+        filtered_validations = {}
+        filtered_validations["Metadata"] = self.completed_valids.current_validations["Metadata"]
+        for validation in filtered:
+            filtered_validations[validation[0]] = {"Outputs": list(validation[1]), "validationName": validation[0]}
+        self.filtered_validations = filtered_validations
+
+    def format_recombinant(self):
```
Comment (RyanAldy) [inline recombinator.py]: Not sure about this method name.  Something more explanatory? Some people may be confused as to what recombinant is?

Comment (RyanKilbane) [reply recombinator.py]: Yeah, that seems reasonable. I changed it to format_recombined_data in latest commit.

**Pull request merged**